src/main/scala/fi/aalto/cs/apluscourses/intellij/utils/ModuleUtils.scala

```diff
@@ -27,15 +27,15 @@ object ModuleUtils {
     FileUtilRt.toSystemIndependentName(ModuleUtilCore.getModuleDirPath(module))
 
   def getModuleOfEditorFile(@NotNull project: Project,
-                            @NotNull dataContext: DataContext): Option[Module] = for {
-    editor <- Option(CommonDataKeys.EDITOR.getData(dataContext))
-    openFile <- Option(FileDocumentManager.getInstance.getFile(editor.getDocument))
-  } yield ModuleUtilCore.findModuleForFile(openFile, project)
+                            @NotNull dataContext: DataContext): Option[Module] =
```

Comment:
a quick test case to ensure the behavior? :nerd_face:

Reply:
That's unfortunately very difficult here due to the calls to `FileDocumenttManager.getInstance.getFile` and `ModuleUtilCore.findModuleForFile`

Reply:
would I otherwise ask? :wink: